app/resources/code.yamil

```diff
@@ -1,74 +1,3 @@
-def calculate(a: int, b: int) -> int = {
```

Comment:
Don't delete what we have in the yamil file please.

Just add the line of the code you want, that in this case is the lambda.